app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, dependent: :destroy
+  has_many :likes, dependent: :destroy
+  has_many :comments, dependent: :destroy
```

Comment:
- [ ] Optional suggestion: 

Instead of doing that with the destroy, I would recommend doing this: it should be author_id or user_id depending on the name used.

```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

Reply:
Hi

Thanks for the review.

Can you explain more? I don't think what you are asking is relevant to what I did there.

My models are perfectly connected with each other. What I did was to make sure if user gets destroyed, relevant posts, comments and likes also get destroy.

Reply:
@ahangarha your models are well connected however you can check my suggestion to get some insight on it works

Reply:
Lemme send you some resources